### ISSUE TITLE
Show traceback in else for error handler

### ISSUE THREAD

**Issue by Phxntxm** [vcokltfre/tutorial#15]
There's one detail I want to be nitpicky about in the error handler section. I'm one of the helpers from the d.py server, so this want comes from seeing what users come in with. It's often that someone sets up an error handler, and doesn't have some kind of `else: do something` in it, leaving them with silenced errors and no idea what to do about it.

This tutorial does have an else, but just says "oh no something went wrong", not giving any actual details about what went wrong. I'd recommend doing this:

```
    traceback_list = traceback.format_exception(type(error), error, error.__traceback__)
    traceback_format = f"```\n{''.join(traceback_list)}```"

    await ctx.send(traceback_format)
```

This will give a nice traceback in the channel the command is sent in with the traceback needed to figure out what's going wrong. 
![image](https://user-images.githubusercontent.com/3664828/126880299-6d640e9d-7188-42cd-ab53-0a5496e7d43e.png)

The reason I don't like the current way it's done is for people who don't yet know that much, their first attempt will be to do `await ctx.send(error)` or something similar, which is simply not enough information to actually figure out the issue more times than not.

**Comment by pedro-anthony** [comment]
I disagree. the error should instead be printed in the console to avoid exceeding the message character limit.

**Comment by Phxntxm** [comment]
> I disagree. the error should instead be printed in the console to avoid exceeding the message character limit.

I've never had a traceback exceed this limit in years of my bot running... That one in the screenshot is half of limit. Regardless that isn't even the point of this issue, the point is tracking it *somehow*

This is a tutorial, not a plug-and-play bot. I don't care how the error is shown, I just want it to be shown somehow
